### DISCUSSION TITLE
How to Set luatex as default?

### DISCUSSION THREAD

**Discussion by jeanneyue** [insightsengineering/teal#1250] [Q&A] [answered]
Hi,

I have installed tinytex and it has luatex. May I know how to generate the pdf with luatex? I got a warning message in the pdf output about the fonts. I don't know how to update the YAML header.

Best,
Jeanne

**Comment by donyunardi** [comment]
Hi @jeanneyue 
Do you have some code example or screenshot of the error that you're seeing?

**Comment by m7pr** [comment, marked as answer]
Hey @jeanneyue - this is the way in which you can change the font in PDF created with rmarkdown where the engine uses LaTeX. https://stackoverflow.com/questions/24421235/changing-font-in-pdf-produced-by-rmarkdown
In the created .rmarkdown file, you need to edit the YAML and knit/render the document
